programs/config_api/src/config_instruction.rs

```diff
@@ -1,26 +1,53 @@
 use crate::id;
 use crate::ConfigState;
+use bincode::serialize;
+use serde_derive::{Deserialize, Serialize};
 use solana_sdk::instruction::{AccountMeta, Instruction};
 use solana_sdk::pubkey::Pubkey;
+use solana_sdk::short_vec;
 use solana_sdk::system_instruction;
 
+#[derive(Debug, Default, Deserialize, Serialize)]
+pub struct ConfigKeys {
+    #[serde(with = "short_vec")]
+    pub keys: Vec<(Pubkey, bool)>,
```

Comment:
Is this the best place to add a comment documenting what that `bool` is for?  I guess if we used a `struct` instead of a tuple it would be self-documenting.  I don't feel super strong about using a struct though